### DISCUSSION TITLE
Custom shortcut in console

### DISCUSSION THREAD

**Discussion by srhrnkn** [posit-dev/positron#5211] [Q&A] [answered]
I've added this to my keybindings.json file and it works, but only in an editor pane. Is there something I should add to that "when" line so that it will work in the console as well?

{
        "key": "Ctrl+n",
        "command": "type",
        "args": { "text": "%in%" },
        "when": "editorTextFocus"
      }

**Comment by juliasilge** [comment, marked as answer]
I would probably use `"when": "editorLangId == r"`, like we do here for the command to insert a pipe:

https://github.com/posit-dev/positron/blob/1cda237653eaad30a3edc45ff47d90bfa3fac24e/extensions/positron-r/package.json#L348-L353

**Reply by srhrnkn** [reply]
works great, thanks!
